replication.go

```diff
@@ -51,7 +51,11 @@ import (
 
 var defaultReplication = 6
 
-const defaultContentSizeLimit = 34_000_000_000
+// maxFilecoinContentSizeLimit = 31GiB512MiB
+const maxFilecoinContentSizeLimit = 33_822_867_456
+
+// minFilecoinContentSizeLimit = 0.24GiB (256 << 10)
```

Comment:
That is 0.25MiB not GiB.
Also I think it's just clearer if it's `256KiB`.

Reply:
shot, thanks @Jorropo